### DISCUSSION TITLE
Allow unnamed groups to be used as presentational-only

### DISCUSSION THREAD

**Discussion by tomparkp** [payloadcms/payload#6701] [Feature Requests & Ideas]
# Documentation Issue

I had a large editor screen that I wanted to organize visually but without impacting the database structure. ~From looking at the docs `name` in group is marked with an asterisk indicating it's required but I found that I was able exclude name and only use `label` to achieve what I was looking for~.

I think it should be noted in the docs as I could see other users wanting the visual benefit of a group without impacting the db. I could also see users not realizing that groups prefix db tables (in Postgres, I'm guessing in Mongo they nest which is logical) so it might be worth calling out in the docs.

## Additional Details

I am using Payload 3.0.0 beta so it's possible this doesn't apply to v2 (I don't think there are v3 docs published so not sure).

**Comment by tomparkp** [comment]
Actually I have to make a correction as it looks like `name` is required for group (seems like the validation wasn't triggering during HMR but when I restarted the server it kicked in.

So this would also be a bit of a feature request: allowing the usage of UI only groups for organizing controls without impacting the database structure.

**Comment by jmikrut** [comment]
Hey @tomparkp — I would love to see the Group have the ability to be presentational-only! Let's create this into a Feature Request and we will get it done for you.

**Comment by JarrodMFlesch** [comment]
This was implemented [here](https://github.com/payloadcms/payload/pull/12318)!
